app/routes/lending/components/LendableObjectEdit.tsx

```diff
@@ -0,0 +1,185 @@
+import { LoadingIndicator, ConfirmModal } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { Field } from 'react-final-form';
+import { useNavigate, useParams } from 'react-router-dom';
+import {
+  createLendableObject,
+  deleteLendableObject,
+  editLendableObject,
+  fetchLendableObject,
+} from 'app/actions/LendableObjectActions';
+import { Content } from 'app/components/Content';
+import {
+  Button,
+  EditorField,
+  Form,
+  SelectInput,
+  SubmitButton,
+  TextInput,
+} from 'app/components/Form';
+import LegoFinalForm from 'app/components/Form/LegoFinalForm';
+import SubmissionError from 'app/components/Form/SubmissionError';
+import { selectAllGroups } from 'app/reducers/groups';
+import { selectLendableObjectById } from 'app/reducers/lendableObjects';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import { roleOptions } from 'app/utils/constants';
+import { spySubmittable } from 'app/utils/formSpyUtils';
+
+type Params = {
+  lendableObjectId: string | undefined;
+};
+
+const LendableObjectEdit = () => {
+  const { lendableObjectId } = useParams<Params>();
+  const isNew = lendableObjectId === undefined;
+
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  if (!isNew) {
+    usePreparedEffect(
+      'fetchLendableObject',
+      () => dispatch(fetchLendableObject(Number(lendableObjectId))),
+      [],
+    );
+  }
+
+  const lendableObject = useAppSelector((state) =>
+    selectLendableObjectById(state, {
+      lendableObjectId,
+    }),
+  );
+
+  const groups = useAppSelector((state) => selectAllGroups(state));
```

Comment:
```suggestion
  const groups = useAppSelector(selectAllGroups);
```
Are all groups needed?